src/hooks/useBalances/useBalances.tsx

```diff
@@ -25,8 +26,26 @@ export const useBalances = (): UseBalancesReturnType => {
       for (const getAdapter of supportedAdapters) {
         const adapter = getAdapter()
         const key = adapter.getType()
-        const address = await adapter.getAddress({ wallet })
-        const balanceResponse = await adapter.getAccount(address)
+
+        let addressOrXpub
+        if (adapter.getType() === 'ethereum') {
+          addressOrXpub = await adapter.getAddress({ wallet })
+        } else if (adapter.getType() === 'bitcoin') {
+          const pubkeys = await wallet.getPublicKeys([
+            {
+              coin: adapter.getType(),
```

Comment:
Completely optional, functionally the same.

```suggestion
        const adapertype = adapter.getType()
        switch (adapterType) {
          case: 'ethereum':
            addressOrXpub = await adapter.getAddress({ wallet })
            break;
          case 'bitcoin':
            const pubkeys = await wallet.getPublicKeys([
              {
                coin: adapterType,
```